components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/util/SAMLSSOUtil.java

```diff
@@ -1432,6 +1432,57 @@ public static boolean validateACS(String tenantDomain, String issuerName, String
         }
 
     }
+
+    /**
+     * Get the Default ACS URL from the service provider config.
+     *
+     * @param tenantDomain
+     * @param issuerName
+     * @return
+     * @throws IdentityException
+     */
+    public static SAMLSSOServiceProviderDO getSPConfig(String tenantDomain, String issuerName) throws
+            IdentityException {
+
+        SSOServiceProviderConfigManager stratosIdpConfigManager = SSOServiceProviderConfigManager.getInstance();
+        SAMLSSOServiceProviderDO serviceProvider = stratosIdpConfigManager.getServiceProvider(issuerName);
```

Comment:
Are you trying to get the service provider in the super-tenant space here?

Reply:
This is applicable for file based sp configurations

Reply:
Is that available for tenants?